Documentation/FAQ.md

```diff
@@ -42,3 +42,91 @@ Here’s a very short summary: Work on what eventually became Ladybird started o
 > _I'd like to have rich text, and we might as well use HTML for that. :^)_
 
 LibHTML eventually became [LibWeb](https://github.com/LadybirdBrowser/ladybird/tree/master/Userland/Libraries/LibWeb) — which in turn eventually grew into being the core part of the browser engine and browser to which, on 4 July 2022, [the name _Ladybird_ was given](https://www.youtube.com/watch?v=X38MTKHt3_I&t=29s).
+
+## What are some things that Ladybird _isn’t_?
+
+- Not a Blink/Chromium shell.
+- Not a WebKit port.
+- Not a Firefox fork.
+
+## What makes Ladybird/[Ladybird Browser Initiative](https://ladybird.org/) different?
```

Comment:
now that this is in FAQ, this bullet has common threads with `What does 'Independent' mean, if you're including third party dependencies?`